### DISCUSSION TITLE
Full quality background/environment texture

### DISCUSSION THREAD

**Discussion by dmkrtz** [google/model-viewer#2902] [Q&A]
Hi, this time not a question but more of a request.

Wouldn't it be technically possible to add an argument to the model-viewer to max-out the background/environment HDR?
Since I'm using this on a PC app only, I'd love to have a full HQ background instead of a blocky one.
I noticed that when selecting Background A and Environment A (or natural/from HDR) the image looks very pixelated.
![screenshot95537_2K](https://user-images.githubusercontent.com/89161020/138235531-b2ac8b1f-804d-45f9-be80-25f36275ac46.png)

Whereas it looks more clean/less pixelated with different textures:
![screenshot95549_2K](https://user-images.githubusercontent.com/89161020/138235517-dda2eb9b-6542-47a8-bf92-ea1873583c8f.png)

I remember I've read it somewhere in the documentation that it's currently limited and just wanted to show a real-time example.
For screenshots like that it would be much appreciated to use the full potential of a loaded HDR.

Thanks in advance!

**Comment by samaneh-kazemi** [comment]
Hi @dmkrtz , 

We already support this. Have you looked at this [example](https://modelviewer.dev/examples/lightingandenv/#skyboxAndEnvrionment)?
